internal/manifests/manifests.go

```diff
@@ -229,6 +234,15 @@ func (m *Manifests) UpdateClusterManifestInternal(ctx context.Context, params op
 		return nil, err
 	}
 
+	if srcFolder != destFolder && srcFileName != destFileName {
```

Comment:
We need to check this if the filename changes, even if the folder name doesn't.

State before: manifests/foo openshift/bar
Rename openshift/bar to openshift/foo

I renamed the file in the same directory, but it's no longer distinct.

Reply:
Good point